### PR TITLE
create url,views

### PR DESCRIPTION
**Pull request by osadolortech**

### CODE REVIEW

File: instagram_clone/settings.py [hunk 1]
```diff
@@ -110,6 +110,10 @@
     'REGISTER_SERIALIZER': 'user.serializers.CustomRegisterSerializer',
 }
 
+REST_AUTH_SERIALIZERS = {
+    'USER_DETAILS_SERIALIZER': 'user.serializers.CustomeUser',
```
Comment (iamr0b0tx) [inline instagram_clone/settings.py]: See spelling

File: instagramapi/permision.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+from rest_framework.permissions import BasePermission,SAFE_METHODS
+
```
Comment (iamr0b0tx) [inline instagramapi/permision.py]: Not a very good name

**Review by iamr0b0tx — CHANGES_REQUESTED** [review]

File: instagramapi/serialzers.py [hunk 3]
```diff
@@ -4,24 +4,29 @@
...
 class LikePostSerilaizer(serializers.ModelSerializer):
     class Meta:
-        models = LikePost
-        fields="__all__"
\ No newline at end of file
+        model = LikePost
+        fields="__all__"
+
+
+class InstapostSrilaizers(serializers.ModelSerializer):
```
Comment (iamr0b0tx) [inline instagramapi/serialzers.py]: spelling

File: instagramapi/userpermission.py [hunk 4]
```diff
@@ -0,0 +1,8 @@
+from rest_framework.permissions import BasePermission,SAFE_METHODS
+
+class CustomPermission(BasePermission):
```
Comment (iamr0b0tx) [inline instagramapi/userpermission.py:3]: not very descriptive name

**Review by iamr0b0tx — CHANGES_REQUESTED** [review]
I will not be available to review PRs anymore. You will need to move on without my reviews from here on out

**Review by iamr0b0tx — CHANGES_REQUESTED** [review]
I will not be available to review PRs anymore. You will need to move on without my reviews from here on out

File: instagramapi/serialzers.py (hunk 3 above)
Comment (osadolortech) [reply instagramapi/serialzers.py]: OK SIR THANK YOU FOR EVERYTHING

**Pull request merged**